packages/genesys-spark-tokens/data/layout & container.json

```diff
@@ -0,0 +1,255 @@
+{
```

Comment:
Is this a valid file name `layout & container.json` ?

Reply:
valid yes. A good idea no :). I will get it changed in future PRs